### DISCUSSION TITLE
厨具配置后仍无法运行计算机

### DISCUSSION THREAD

**Discussion by lilycoco111** [hjenryin/BCJH-Metropolis#204] [Q&A] [answered]
![image](https://github.com/hjenryin/BCJH-Metropolis/assets/138025413/ceaac300-2560-4683-a77f-7a10119463f6)

**Comment by lilycoco111** [comment]
![image](https://github.com/hjenryin/BCJH-Metropolis/assets/138025413/5da6a66a-004d-4d20-94e4-8aa211a4b8a3)
![image](https://github.com/hjenryin/BCJH-Metropolis/assets/138025413/a4553bfd-2674-4f81-ad24-41c1a8bc45c7)

**Comment by hjenryin** [comment, marked as answer]
是不是没解压
